tests/test_smab.py

```diff
@@ -802,3 +829,82 @@ def test_smab_mo_cc_from_state(state):
     # Ensure get_state and from_state compatibility
     new_smab = globals()[smab.get_state()[0]].from_state(state=smab.get_state()[1])
     assert new_smab == smab
+
+
+########################################################################################################################
+
+
+# Smab with epsilon-greedy super strategy
+
+
+@given(
+    st.integers(min_value=1),
+    st.integers(min_value=1),
+)
+def test_can_instantiate_epsilon_greddy_smab_with_params(a, b):
+    s = SmabBernoulli(
+        actions={
+            "action1": Beta(n_successes=a, n_failures=b),
+            "action2": Beta(n_successes=a, n_failures=b),
+        },
+        epsilon=0.1,
+        default_action="a1",
```

Comment:
here the initialization of this SmabBernoulli should raise an error: `"a1" is not in the actions set `

Reply:
Right! Gr8 catch!
I added relevant functionality under _get_valid_actions and a new root_validator for default_action under the BaseMAB.